yb_migrate/src/migration/oracle.go

```diff
@@ -196,26 +196,27 @@ func Ora2PgExportDataOffline(ctx context.Context, source *srcdb.Source, exportDi
 	exportDataCommand := exec.Command("/bin/bash", "-c", exportDataCommandString)
 	log.Infof("Executing command: %s", exportDataCommandString)
 
-	stdOutFile, err := os.OpenFile(exportDir+"/temp/export-data-stdout", os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
-	if err != nil {
-		utils.ErrExit("Error while opening export data output file: %v", err)
-	}
-	defer stdOutFile.Close()
+	// stdOutFile, err := os.OpenFile(exportDir+"/temp/export-data-stdout", os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
+	// if err != nil {
+	// 	utils.ErrExit("Error while opening export data output file: %v", err)
+	// }
+	// defer stdOutFile.Close()
 
-	stdErrFile, err := os.OpenFile(exportDir+"/temp/export-data-stderr", os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
-	if err != nil {
-		utils.ErrExit("Error while opening export data error file: %v", err)
-	}
-	defer stdErrFile.Close()
+	// stdErrFile, err := os.OpenFile(exportDir+"/temp/export-data-stderr", os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
+	// if err != nil {
+	// 	utils.ErrExit("Error while opening export data error file: %v", err)
+	// }
+	// defer stdErrFile.Close()
 
-	exportDataCommand.Stdout = stdOutFile
-	exportDataCommand.Stderr = stdErrFile
+	// exportDataCommand.Stdout = stdOutFile
+	// exportDataCommand.Stderr = stdErrFile
 
-	err = exportDataCommand.Start()
+	stdout, err := exportDataCommand.CombinedOutput()
```

Comment:
@rahulb-yb have you tested these changes?
You have changed `Start()` call to `CombinedOutput()`
This is going to break code/tool i believe

Start() - will start the execution of command and don't wait command to exit/complete

So with CombinedOutput(), your current goroutine will stay at this point and line 221 will not be executed till the end which in turn will cause the progress bar and everything to stall.

Reply:
I was reviewing commit by commit.
I think at the end you have corrected this.

Reply:
yeah, I was attempting to understand and compare at first with main, so I temporarily had this as an initial commit

Reply:
> nit: add complete path for yb_migrate.log fmt.Printf("pg_dump failed to export data with error: %v. Refer to'%s/yb_migrate.log' for further details.", err, exportDir)

sorry, I missed a space after `Refer to` 
fmt.Printf("pg_dump failed to export data with error: %v. Refer to '%s/yb_migrate.log' for further details.", err, exportDir)